_docs/reference/release-notes.md

```diff
@@ -88,5 +88,10 @@ identity provisioning. This agent runs on each node (VM / physical machine) and
 - **Persistent CA Key/Certificate Storage**. Istio CA now stores signing key/certificates in
 persistent storage to facilitate CA restarts. 
 
+### Known Issues
```

Comment:
maybe we should also list here that with auth enabled you can't talk to k8s api server or non istio services
(and that for health checks you need to use a custom curl command)

Reply:
these are already documented in security FAQ, but I did add non istio service as that is pretty major.  talking to K8S API server isn't that common...